### PR TITLE
Update verified parameter – group 0

### PR DESCRIPTION
**Pull request by gtmgianni**
Converts verified comment to verified parameter of `url`

### CODE REVIEW

File: Casks/brave-browser-nightly.rb [hunk 1]
```diff
@@ -2,12 +2,11 @@
   version :latest
   sha256 :no_check
 
-  # updates.bravesoftware.com/ was verified as official when first introduced to the cask
   url do
     require "open-uri"
     appcast = "https://updates.bravesoftware.com/sparkle/Brave-Browser/nightly/appcast.xml"
     URI(appcast).read[/enclosure url="([^"]+.dmg)"/, 1]
-  end
+  end, verified: "updates.bravesoftware.com/"
```
Comment (vitorgalvao) [inline Casks/brave-browser-nightly.rb:9]: No idea if this will work.

Comment (gtmgianni) [reply Casks/brave-browser-nightly.rb:9]: I was trying to figure out how to do this – not sure what the correct syntax is.

Comment (vitorgalvao) [reply Casks/brave-browser-nightly.rb:9]: I’m unsure it `url do` can handle parameters. If this fails, I’ll know who to ping.

Comment (reitermarkus) [reply Casks/brave-browser-nightly.rb:9]: It should work with

```
[URI(appcast).read[/enclosure url="([^"]+.dmg)"/, 1], {verified: "updates.bravesoftware.com/"}]
```

But `url do` blocks could return anything, so it doesn't really make sense to verify them.

Comment (vitorgalvao) [reply Casks/brave-browser-nightly.rb:9]: @claui Is there any way for `url do` to use [the parameters](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#additional-https-url-parameters)?

Comment (vitorgalvao) [reply Casks/brave-browser-nightly.rb:9]: @claui Please ignore, I hadn’t seen that @reitermarkus had already posted and answer.

Comment (vitorgalvao) [reply Casks/brave-browser-nightly.rb:9]: > But `url do` blocks could return anything, so it doesn't really make sense to verify them.

But they still fail (or used to fail) `audit` without the comment.

Comment (reitermarkus) [reply Casks/brave-browser-nightly.rb:9]: We can make the audit ignore it for `url do` blocks.

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**